packages/pockethost.io/src/util/stores.ts

```diff
@@ -1,3 +1,22 @@
 import { writable } from 'svelte/store'
 
-const instanceCreationWidgetName = writable(0)
+export const globalUserData = writable({})
```

Comment:
Consider filling out the generic here so the type passed to globalUserData will be strongly typed.